packages/common/src/paths.tsx

```diff
@@ -0,0 +1,26 @@
+export const PATH_HOME = '/';
+
+export const PATH_MEDIA = '/m/:id/*';
+export const PATH_PLAYLIST = '/p/:id/*';
+export const PATH_LEGACY_SERIES = '/s/:id/*';
+
+export const PATH_SEARCH = '/q/*';
+export const PATH_ABOUT = '/o/about';
+
+export const PATH_USER_BASE = '/u';
```

Comment:
Maybe we could use only one constant for main routes?

```
const PATH = {
    HOME: '/',
    ....
}
```

Reply:
What do you think about using one constant for routes?

Reply:
We've discussed this and concluded that this is an improvement (although minor considering this PR). We have created a ticket to change this in the next sprint. Is that OK?